external/DIA2Dump/dia2dump.cpp

```diff
@@ -10,7 +10,7 @@
 // Debug Interface Access SDK API.
 //
 
-#include "Dia2Dump.h"
+#include "dia2dump.h"
```

Comment:
This was necessary to make it compile for windows?

Reply:
not really :) I was trying to bring consistency to includes..